pdns/pdns_recursor.cc

```diff
@@ -2516,9 +2516,16 @@ static void doStats(void)
 
 static void houseKeeping(void *)
 {
-  static thread_local time_t last_rootupdate, last_prune, last_secpoll;
+  static thread_local time_t last_rootupdate, last_prune, last_secpoll, last_trustAnchorUpdate;
   static thread_local int cleanCounter=0;
   static thread_local bool s_running;  // houseKeeping can get suspended in secpoll, and be restarted, which makes us do duplicate work
+
+  auto luaconfsLocal = g_luaconfs.getLocal();
+  if (!luaconfsLocal->trustAnchorFileInfo.second.empty() && luaconfsLocal->trustAnchorFileInfo.first != 0) {
+    // Loading the config updated this, so we fake the update time here
```

Comment:
I feel like I'm clearly missing something, but isn't `last_trustAnchorUpdate` reset every time we enter `houseKeeping`?